packages/core/content-type-builder/admin/src/components/FormModal/reducer.js

```diff
@@ -220,6 +221,46 @@ const reducer = (state = initialState, action) =>
         draftState.initialData = action.data;
         break;
       }
+      case actions.SET_CUSTOM_FIELD_DATA_SCHEMA: {
```

Comment:
Could this be an opportunity for a new tracker event? one that makes sure the custom field attribute was actually added

Reply:
This actually won't make sure the custom field was added. This just sets the initial object shape when selecting a custom field. Which in all cases will be something like: `{name: null, type: <type>}` I think